app/Locale/French.php

```diff
@@ -51,6 +51,7 @@ public function get_locale(): array
 			'DEFAULT_LICENSE' => 'License par defaut pour les nouveaux ajouts :',
 			'SET_LICENSE' => 'Sélectioner une license',
 			'SET_OVERLAY_TYPE' => 'Sélectioner le type d’Overlay',
```

Comment:
```suggestion
			'SET_OVERLAY_TYPE' => 'Sélectionner le type d’Overlay',
```
typo ;p

Reply:
There were a few more of the same type 😉